docs/developer-guide/get-started/emt-build-and-deploy.md

```diff
@@ -0,0 +1,373 @@
+     The file will be located in the `./standalone-node/installation-scripts/out` directory.
+
+
+4. Copy the contents of `standalone-installation-files.tar.gz`:
+
+   Since the `./standalone-node/installation-scripts/out` folder is read only, you need to
+   select a folder to which you will extract the required files and then edit them.
+   For example, you can create the `standalone-installation-files` folder in the
+   `\home` directory:
+
+   ```bash
+   mkdir ~\standalone-installation-files
+   ```
+
+   Extract the contents to the selected folder:
+
+   ```bash
+   tar -xzf ./standalone-node/installation_scripts/out/standalone-installation-files.tar.gz -C ~/standalone-installation-files
+   ```
+
+   The extracted files are as follows:
+
+   ```text
+   bootable-usb-prepare.sh
+   write-image-to-usb.sh
+   config-file
+   usb-bootable-files.tar.gz
+   edgenode-logs-collection.sh
+   standalone-vm-setup.sh
+   download_images.sh
+   user-apps
+   ```
+
+   Navigate to the folder with the extracted installation files:
+
+   ```bash
+   cd ~/standalone-installation-files
+   ```
+
+   Download the Kubernetes artifacts (container images and manifest files) by
+   executing the `./download_images.sh` script. For the default non-real-time image,
+   use the `NON-RT` parameter.
+
+   ```bash
+   sudo ./download_images.sh NON-RT
+   ```
+
+   > **Note:** By default, the script will only pull basic kubernetes artifacts to create a
+     single node cluster. If you want to use EMT image with desktop virtualization features,
+     use the `DV` parameter instead of `NON-RT`.
+
+   Update the `config-file` with your deployment-specific settings.
+   This configuration file is used to provision the edge node during
+   its initial boot and should include the following parameters:
+
+   - **Proxy settings** - when the edge node requires a proxy to access external networks.
+   - **SSH key** - the public SSH key (typically your `id_rsa.pub`) from your Linux
+     development system to enable passwordless SSH access to the edge node.
+   - **User credentials** - the username and password for the primary user account on the edge node.
+   - **Cloud-init customization** - optional, user-defined `cloud-init` configurations for advanced setup requirements.
+
+     For the default non-RT image, a basic Kubernetes installation is done automatically.
+
+     > **Note:** For deployments requiring desktop virtualization features,
+     > refer to the [desktop virtualization image guide](https://github.com/open-edge-platform/edge-microvisor-toolkit-standalone-node/blob/main/standalone-node/docs/user-guide/desktop-virtualization-image-guide.md) in the
+     > `user-guide` directory. This document provides reference `cloud-init` configurations that can be
+     > tailored to your specific deployment needs.
+
+   - **HugePages configuration** - set when the workloads require HugePages.
```

Comment:
No link or refernce how to enable huge pages support here... is that intended. It seems like something is missing.

Reply:
It reflects how it's described in the [user guide](https://github.com/sgolebiewski-intel/edge-microvisor-toolkit-standalone-node/blob/main/standalone-node/docs/user-guide/get-started-guide.md#13-prepare-the-bootable-usb-drive) in the standalone-node repository;